fuse_models/src/unicycle_2d_ignition.cpp

```diff
@@ -230,6 +230,12 @@ void Unicycle2DIgnition::process(const geometry_msgs::PoseWithCovarianceStamped&
   // Tell the optimizer to reset before providing the initial state
   if (!params_.reset_service.empty())
   {
+    // Wait for the reset service
+    while (!reset_client_.waitForExistence(ros::Duration(30.0)) && ros::ok())
```

Comment:
10s does seem less extreme, if you want to make that change.

Reply:
Changed to `10.0`s